src/mysql/migrate.test.ts

```diff
@@ -34,7 +33,7 @@ describe('Migrate', () => {
 
   beforeAll(async () => {
     const mySqlServer = new MySQLServer({ mysqlBaseDir: 'mysql-context', mysqldPath })
-    console.log(await mySqlServer.getTimings())
+    // console.log(await mySqlServer.getTimings())
```

Comment:
Is the commented logging intentional?

Reply:
Yes, I don't think it's useful to always have in the test output.